src/OrleansCodeGenerator/RoslynCodeGenerator.cs

```diff
@@ -317,36 +317,31 @@ private static GeneratedSyntax GenerateForAssemblies(List<Assembly> assemblies,
 
             var members = new List<MemberDeclarationSyntax>();
 
+            // If any KnownAssemblies have been specified, include them during code generation.
+            var knownAssemblies =
+                assemblies.SelectMany(_ => _.GetCustomAttributes<KnownAssemblyAttribute>())
+                    .Select(_ => _.Assembly)
```

Comment:
nit: avoid using "_" as the variable name, if you actually expect to use it in the lambda expression